### ISSUE TITLE
Identify unused resources

### ISSUE THREAD

**Issue by 0xdabbad00** [duo-labs/cloudmapper#461]
In addition to #372 for security groups, find other resources in the account that are unused.

**Comment by 0xdabbad00** [comment]
Find unused volumes: https://blog.andreev.it/?p=3009

**Comment by 0xdabbad00** [comment]
Find unused ENI's:

```
aws ec2 describe-network-interfaces
	--filters Name=status,Values=available
```

**Comment by 0xdabbad00** [comment]
This concept is what Janitor Monkey does. See what it cleans up: https://github.com/Netflix/SimianArmy/tree/master/src/main/java/com/netflix/simianarmy/aws/janitor

**Comment by 0xdabbad00** [comment]
Looking at Janitor Monkey, it has extra logic: https://github.com/Netflix/SimianArmy/blob/dd2ff51cd0b52d501e0c918fecb4f0e1cfb8e2e5/src/main/java/com/netflix/simianarmy/aws/janitor/rule/ami/UnusedImageRule.java#L70

For an AMI, it looks to see if this is a base image, used to create other images.  It also checks some threshold number of days for when this was last used.  I'm not sure how it determines that, but based on some of the other code, it seems to use tagging and possibly SimpleDB to keep track of things like this.

**Comment by 0xdabbad00** [comment]
The rules for Janitor Monkey are here: https://github.com/Netflix/SimianArmy/tree/dd2ff51cd0b52d501e0c918fecb4f0e1cfb8e2e5/src/main/java/com/netflix/simianarmy/aws/janitor/rule

I don't think I can base any work off of their concepts though as the code is integrated with a lot of other things the Simian army does.  Some is also too opinionated, for example, an EC2 instance is marked as orphaned if it is not part of an auto-scaling group or OpsWorks.

**Comment by 0xdabbad00** [comment]
Find unused EBS volumes:
```
aws ec2 describe-volumes
	--filters Name=status,Values=available
```

**Comment by 0xdabbad00** [comment]
Resolved by #484 484
